src/Tooltip/Tooltip.js

```diff
@@ -102,7 +102,19 @@ var TooltipContent = kind({
 	*/
 	uppercaseChanged: function (was, is) {
 		this.contentChanged();
-	}
+	},
+
+	// Accessibility
+
+	/**
+	* @private
+	*/
+	ariaObservers: [
+		{path: ['content'], method: function () {
+			var content = this._content;
```

Comment:
I think it's okay to use 'this.content' since 'ariaObservers' are triggered prior than 'somethingChanged'.
Could you check timing?